config/app.php

```diff
@@ -149,7 +149,7 @@
     |
     */
 
-    'trust_proxies' => env('APP_TRUST_PROXIES'),
+    'trust_proxies' => env('APP_TRUSTED_PROXIES'),
```

Comment:
It needs to be fixed as
```suggestion
    'trust_proxies' => env('APP_TRUSTED_PROXIES', env('APP_TRUST_PROXIES')),
```